### ISSUE TITLE
react17 dependency error

### ISSUE THREAD

**Issue by kgh-85** [markablov/js-4004#24]
tldr; to fix the error:

- In `/package.json` replace `"react-loading-overlay": "^1.0.1", ` with `"@ronchalant/react-loading-overlay": "^1.0.1", `
- In `/src/components/BusyOverlay.js` replace `import LoadingOverlay from 'react-loading-overlay';` with `import LoadingOverlay from '@ronchalant/react-loading-overlay';`

Explanation:

I was not able to run the application with a fresh npm / node.js install on Windows because of the following dependency error:

> npm ERR! code ERESOLVE
> npm ERR! ERESOLVE unable to resolve dependency tree
> npm ERR!
> npm ERR! While resolving: i40xx-emu@1.0.0
> npm ERR! Found: react@17.0.2
> npm ERR! node_modules/react
> npm ERR!   react@"^17.0.2" from the root project
> npm ERR!
> npm ERR! Could not resolve dependency:
> npm ERR! peer react@"^0.14 || ^15.0.0-rc || ^15.0 || ^16.0.0 || ^16.0" from react-loading-overlay@1.0.1
> npm ERR! node_modules/react-loading-overlay
> npm ERR!   react-loading-overlay@"^1.0.1" from the root project

If I got it right, the root cause is that the project wants react@17.0.2 but react-loading-overlay is not available in react@17

I then found:
https://www.npmjs.com/package/@ronchalant/react-loading-overlay

which states:

> "This package is generated from a fork of [Derrick Pelletier](https://github.com/derrickpelletier)'s original [react-loading-overlay](https://github.com/derrickpelletier/react-loading-overlay). I needed a version that was compatible with React 17, and so generated this package with minimal changes from his original version."
